compiler/rustc_metadata/src/rmeta/decoder/cstore_impl.rs

```diff
@@ -423,6 +423,8 @@ provide! { tcx, def_id, other, cdata,
         syms
     }
 
+    get_default_impl_equivalent => { cdata.get_default_impl_equivalent(def_id.index) }
```

Comment:
pretty sure this is just `{ table }`

```suggestion
    get_default_impl_equivalent => { table }
```